pytorch/training/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,318 @@
+FROM nvidia/cuda:11.1.1-base-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.8.5
+ARG CUBLAS_VERSION=11.3.0.106
+ARG OPEN_MPI_PATH=/opt/amazon/openmpi
+ARG EFA_PATH=/opt/amazon/efa
+
+ARG CUDA_HOME=/usr/local/cuda
+ARG CONDA_PREFIX=/opt/conda
+ARG METIS=metis-5.1.0
+ARG RMM_VERSION=0.15.0
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
+ENV LD_LIBRARY_PATH="/opt/conda/lib:${LD_LIBRARY_PATH}"
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+ENV TORCH_CUDA_ARCH_LIST="3.7 5.0 7.0+PTX 8.0"
+ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
+ENV CUDNN_VERSION=8.0.5.39
+ENV NCCL_VERSION=2.7.8
+ENV HOROVOD_VERSION=0.21.3
+ENV EFA_VERSION=1.11.2
+ENV OMPI_VERSION=4.1.1
+ENV BRANCH_OFI=1.1.1
+ENV DGLBACKEND=pytorch
+ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
+ENV MANUAL_BUILD=0
+ENV RDMAV_FORK_SAFE=1
+ENV TZ=America/New_York
```

Comment:
Could you please explain why this is necessary on the PT 1.9 DLC, and not on the PT 1.8 DLC?

Also, would we be able to get away with only the `ENV DEBIAN_FRONTEND=noninteractive` command?

A DLC is meant to be used in multiple time-zones around the world, and if this affects the behavior in time-zones other than New York, then we should be more apprehensive about this change.

Reply:
We are able to  get away with only the `ENV DEBIAN_FRONTEND=noninteractive` command? Updated dockerfile to remove region configuration, able to build successfully after that.